frontend/src/components/TaskView/FutureView/FutureViewTask.tsx

```diff
@@ -53,9 +57,20 @@ function FutureViewTask({
 }: Props): ReactElement | null {
   const isSmallScreen = useMappedWindowSize(({ width }) => width <= 840);
 
+  const [{ openedGlobal }, setOpened] = useState<MyState>({
+    openedGlobal: false,
+  });
```

Comment:
1. the name `openedGlobal` is confusing
2. No need to put it in object. you can do `const [openedGlobal, setOpenedGlobal] = useState(false);`